GSA-TTS/FAC
# Add FMCSA to the List of Valid Cluster Names

# Add FMCSA to the List of Valid Cluster Names

Issue: https://github.com/GSA-TTS/FAC/issues/3490

## Changes:
1. Add FMCSA to the cluster names CSV. It's at the end, but sorts to the correct row alphabetically. 
2. Regenerate the workbooks and json by way of a `make all` in the schemas folder.
    * This means some previous code changes are represented here in the final workbooks.

## How to test:
1. Attempt to upload a federal awards workbook with cluster name `Federal Motor Carrier Safety Assistance (FMCSA)`*. Verify that it fails on the cluster name. 
2. Switch to this branch and run normally. Verify that the same workbook now passes on the cluster name. 

* An existing workbook with this cluster name can be pulled from ZD. 

## PR checklist: submitters


## PR checklist: reviewers


The larger the PR, the stricter we should be about these points.
